presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -129,34 +125,25 @@ public void tearDown()
         scheduledExecutor.shutdownNow();
     }
 
-    @DataProvider(name = "hashJoinTestValues")
-    public static Object[][] hashJoinTestValuesProvider()
+    @DataProvider(name = "hashEnabledValues")
+    public static Object[][] hashEnabledValuesProvider()
     {
-        // parallelBuild, probeHashEnabled, buildHashEnabled, and isDictionaryProcessingJoinEnabled respectively
         return new Object[][] {
-                {true, true, true, true},
-                {true, true, true, false},
-                {true, true, false, true},
-                {true, true, false, false},
-                {true, false, true, true},
-                {true, false, true, false},
-                {true, false, false, true},
-                {true, false, false, false},
-                {false, true, true, true},
-                {false, true, true, false},
-                {false, true, false, true},
-                {false, true, false, false},
-                {false, false, true, true},
-                {false, false, true, false},
-                {false, false, false, true},
-                {false, false, false, false}};
+                {true, true, true},
+                {true, true, false},
+                {true, false, true},
+                {true, false, false},
+                {false, true, true},
+                {false, true, false},
+                {false, false, true},
+                {false, false, false}};
     }
 
-    @Test(dataProvider = "hashJoinTestValues")
-    public void testInnerJoin(boolean parallelBuild, boolean probeHashEnabled, boolean buildHashEnabled, boolean isDictionaryProcessingJoinEnabled)
+    @Test(dataProvider = "hashEnabledValues")
```

Comment:
`hashJoinTestValues` is actually better name, as this includes `parallelBuild` param